internal/renderers/femtovg/fonts.rs

```diff
@@ -484,14 +407,7 @@ impl FontCache {
         _primary_font: &LoadedFont,
         _reference_text: &str,
     ) -> Vec<SharedString> {
-        sharedfontdb::FONT_DB.with(|db| {
-            db.borrow()
-                .fontconfig_fallback_families
-                .iter()
-                .filter(|&family_name| self.is_known_family(family_name))
-                .map(|family_name| family_name.into())
-                .collect()
-        })
+        Vec::new()
```

Comment:
I think this function (including all of its "overloads") should be replaced with use of [`Collection::fallback_families()`](https://docs.rs/fontique/0.5.0/fontique/struct.Collection.html#method.fallback_families). Is this something you were considering for this PR or a follow-up?

Reply:
Yes I believe we can. That's not a piece of code that I'm super familiar with though. I'd rather leave it for a followup if that's acceptable.